PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureComboBox.java

```diff
@@ -158,6 +156,10 @@ private void refreshValue() {
                 super.setSelectedIndex(0);
             }
         } else {
+            final Object selected = this.getSelectedItem();
+            if (Objects.equals(selected, this.value) || (this.value instanceof ItemReference && ((ItemReference<?>) this.value).is(selected))) {
+                return;
```

Comment:
Will it break the operation of set value with `ItemReference`? As line165 `items.stream().filter(i -> ((ItemReference<?>) this.value).is(i)).findFirst().ifPresent(this::setValue);` will call setValue with the item reference result, which may return here during setValue